test/test_pairs_classifiers.py

```diff
@@ -545,11 +539,12 @@ def breaking_fun(**args):  # a function that fails so that we will miss
   assert str(raised_error.value) == expected_msg
 
 
-@pytest.mark.parametrize('estimator, build_dataset', pairs_learners,
-                         ids=ids_pairs_learners)
+@pytest.mark.parametrize('estimator, build_dataset', pairs_learners_m,
+                         ids=ids_pairs_learners_m)
 def test_accuracy_toy_example(estimator, build_dataset):
   """Test that the accuracy works on some toy example (hence that the
-  prediction is OK)"""
+  prediction is OK). This test is designed for Mahalanobis learners only,
```

Comment:
maybe we can easily design a similar test for bilinear learners, eg by using half the similarity score as the threshold?